modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/internal/classloader/AbstractArtifactClassLoaderConfigurationAssembler.java

```diff
@@ -72,6 +85,13 @@ public ClassLoaderModel createClassLoaderModel() {
     return classLoaderConfigurationBuilder.build();
   }
 
+  private void includePatches(ClassLoaderModelBuilder classLoaderConfigurationBuilder) {
+    if (!isMuleFramework()) {
```

Comment:
wouldn't `getArtifactPatches()` return an empty collection in muleFramework mode?